SonarQube.Bootstrapper/BuildAgentUpdater.cs

```diff
@@ -58,6 +58,12 @@ public bool TryUpdate(string hostUrl, string targetDir, ILogger logger)
                         return false;
                     }
 
+                    if (e.Status == WebExceptionStatus.ConnectFailure)
```

Comment:
There was a small bug in how we handle connection failures - NameResolutionFailure is thrown when the URL cannot be resolved against a DNS, but URLs such as localhost or IP addresses fail with ConnectionFailed